contributing.md

```diff
@@ -15,17 +15,17 @@ This file contains notes for potential contributors to GPflow, as well as some n
 
 ## Project scope
 
-With GPflow, we aim to make an extensible library for Gaussian processes which makes building complex models easy. In order to do this, we aim to make GPflow a complete library for doing inference and prediction in sophisticated ways (focussing mainly on variational inference) for *single layer models* only. In order to allow more complicated models to be implemented, we also provide functionality for latent / uncertain inputs. Other models, like deep GPs, can be implemented in their own repository by using GPflow as a dependency. We choose to limit the scope deliberately in order to ensure a high quality codebase.
+With GPflow, we aim to make an extensible library for Gaussian processes which makes building complex models easy. In order to do this, we aim to make GPflow a complete library for doing inference and prediction in sophisticated ways (focussing mainly on variational inference) for *single layer models* only. In order to allow more complicated models to be implemented, we also provide functionality for latent / uncertain inputs. Other models, like deep GPs, can be implemented in their own repository by using GPflow as a dependency. We choose to limit the scope deliberately in order to ensure a high-quality codebase.
 
-We welcome contributions to GPflow. If you would like to contribute a feature, please raise discussion via a GitHub issue, to discuss the suitability of the feature within GPflow. If the feature is outside the envisaged scope, we can still link to a separate project in our Readme. Large features also make it onto the [roadmap](roadmap.md).
+We welcome contributions to GPflow. If you would like to contribute a feature, please raise discussion via a GitHub issue, to discuss the suitability of the feature within GPflow. If the feature is outside the envisaged scope, we can still link to a separate project in our Readme. <!-- Large features also make it onto the [roadmap](roadmap.md). -->
```

Comment:
We don't currently have a roadmap.md file. Should we have one ?

Reply:
i think github's doing just fine for shorter term ideas. would a roadmap be for longer term ideas? I reckon they could be captured in github tickets for now. But I don't know what the longer term research ideas are. What do the others think?

Reply:
if we don't go with a roadmap.md, we should remove it from this doc

Reply:
I think a roadmap would be useful for putting things into perspective, instead of an unsorted list of issues... which is why I didn't delete it outright

Reply:
oooh ... i read it backwards. I thought you were removing comments

Reply:
resolving comment and adding a ticket to the new board

Reply:
I've removed it from here for now